roles/openstack-network/templates/neutron_external_interface.cfg

```diff
@@ -0,0 +1,9 @@
+allow-hotplug {{ neutron_external_interface }}
+iface {{ neutron_external_interface }} inet static
+{% for subnet in neutron.subnets %}
+{% if subnet.name == 'external' %}
+    address {{ subnet.cidr|ipaddr('net')|ipaddr('1')|ipaddr('address') }}
+    netmask {{ subnet.cidr|ipaddr('net')|ipaddr('netmask') }}
+{% endif %}
+{% endfor %}
+    up sysctl net.ipv4.conf.all.forwarding=1
```

Comment:
We need include static routes for additional external subnets and any internal subnets if SNAT is disabled on the Neutron router.